packages/webpack/src/plugins/nx-webpack-plugin/nx-app-webpack-plugin-options.ts

```diff
@@ -215,6 +219,10 @@ export interface NxAppWebpackPluginOptions {
    * Use tsconfig-paths-webpack-plugin to resolve modules using paths in the tsconfig file.
    */
   useTsconfigPaths?: boolean;
+  /**
+   * The implementation of the SASS compiler to use. Can be either `sass` or `sass-embedded`. Defaults to `sass-embedded`.
+   */
+  sassImplementation?: 'sass' | 'sass-embedded';
```

Comment:
Add a `TODO(v22)` so we can remove this an use `sass-embedded` by default. And also migrate existing users.